config/ConfAssistant-template.php

```diff
@@ -205,7 +205,7 @@ class ConfAssistant
     /**
      * Determines if DiscoJuice keywords should be used in the discovery service
      * The keywords contain other language variants of the IdP name making it
-     * easier to kollow keyboard search. Turning this oprion on will add
+     * easier to follow keyboard search. Turning this option on will add
```

Comment:
Not sure about this one : `kollow` → `follow`

Reply:
Yes, it should be "follow".